SLI_README.md

```diff
@@ -0,0 +1,155 @@
+# SLI (Service Level Indicators)
+
+
+### Number of tables which have't have backups after X days since last change.  
+
+This metric is measuring number of tables which are not backed up since X days from last observed change. 
+
+To measure it, it is required to have [GCP Census](https://github.com/ocadotechnology/gcp-census) application scanning all data which should be backed up 
+and also turning on daily datastore exports. 
```

Comment:
Link to [cloud datastore export](https://github.com/ocadotechnology/bbq/blob/master/SETUP.md#cloud-datastore-export), please

Reply:
done